src/entity/Alerts.ts

```diff
@@ -52,7 +52,7 @@ export const ALERTS_DEFAULT: Omit<Alerts, 'userId' | 'flags'> = {
   myFeed: null,
   companionHelper: true,
   lastChangelog: new Date(),
-  lastBanner: new Date('2023-02-05 12:00:00'), // Has to be in the past to accommodate new users
+  lastBanner: new Date(),
```

Comment:
Changing this would cause new users to not see an active banner, wouldn't it?

Reply:
For anonymous users (viewing post page), I think so, yeah. That's why I was also thinking if we should store some cache value related to it in Redis. https://github.com/dailydotdev/daily-api/pull/1528#discussion_r1388003150

Reply:
Nevermind, I checked the boot, they always had `banner` property to false, anyway. So I think it would not matter anymore since it is only to be concerned for anonymous users.